src/pages/Favourites.tsx

```diff
@@ -0,0 +1,13 @@
+import { Box } from "@mui/material"
+import Forecast from "../components/Forecast/Forecast"
+const Favourites = () => {
+    return (
+        <Box m={5} display={'grid'} gridTemplateColumns={'1fr 1fr 1fr 1fr 1fr'}>
```

Comment:
repeat
https://developer.mozilla.org/en-US/docs/Web/CSS/repeat
```suggestion
        <Box m={5} display={'grid'} gridTemplateColumns={'repeat(5,1fr)'}>
```

Reply:
wow cool